### PR TITLE
Reorganize deployingTheSite page

### PR DESCRIPTION
**Pull request by raysonkoh**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

Currently, the `deployingTheSite` page is quite cluttered, with some amount content repetition. The purpose of this PR is to streamline the content in `deployingTheSite.md`, to facilitate PR #1487 .

**Overview of changes:**
- Moved content for using a specific version of Markbind into a separate `Tips & Tricks` section.
- Update `Deploying to Github Pages` section to be less nested

**Anything you'd like to highlight / discuss:**
NIL

**Testing instructions:**
`npm run test`

**Proposed commit message: (wrap lines at 72 characters)**
Reorganize deploying the site page

---

**Checklist:** :ballot_box_with_check:

- [x] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No blatantly unrelated changes
- [ ] Pinged someone for a review!

### CODE REVIEW

File: docs/userGuide/deployingTheSite.md [hunk 1]
```diff
@@ -310,31 +308,14 @@ Now your site will be deployed on Netlify at the given address specified after d
 Additionally, **when contributors make a pull request to your GitHub repo, you can _preview_ the updated site** at the bottom of the pull request by clicking on `details` link in the PR:
 <include src="screenshot.md" boilerplate var-alt="Preview deploy" var-file="netlifyPreview4.png" inline />
 
-{{ icon_info }} Note that when Netlify is set up as explained above, it will use the latest version of MarkBind which may be a later version than the one you use locally. If you want Netlify to use a specific version of Netlify, follow the instructions in the box below:
-
-<box>
+## Relevant Tips & Tricks
 
-##### Setting up Netlify to use a specific version of MarkBind
+<panel header="Configuring Online Deployment platforms to use specific MarkBind version" expand-headerless>
```
Comment (ang-zeyu) [inline docs/userGuide/deployingTheSite.md:313]: would 
```suggestion
<panel header="Configuring Online Deployment platforms to use specific MarkBind version" type="seamless" expand-headerless>
```
be more consistent?

**Review by ang-zeyu — COMMENTED** [review]
Lgtm generally 👍 

On the same lines as reducing clutter, could also:
- add some blank lines between images and preceding text where appropriate
- shrink some of the images, which are unnecessarily large (the netlify ones mainly)

File: docs/userGuide/deployingTheSite.md (hunk 1 above)
Comment (raysonkoh) [reply docs/userGuide/deployingTheSite.md]: Yup it looks more consistent. Thanks for the suggestion!

**Comment by raysonkoh** [comment]
Thanks for the PR review @ang-zeyu ! I have updated it according to your comments.

File: docs/userGuide/tipsAndTricks.md [hunk 2]
```diff
@@ -69,3 +69,48 @@ When you use links or triggers, you may encounter a situation where an unwanted
...
+
+<span id="useSpecificMarkbind">
+
+##### :fas-info: Configuring Online Deployment platforms to use specific MarkBind version
+
+**Configuring CI platforms to use specific MarkBind version**
+
+When the [default CI configuration for deployment](deployingTheSite.html#using-ci-platforms) is used, the latest version of MarkBind will be used in the CI workflows. This may be a later version of MarkBind than the one you use locally.
+
+* If you want to specify a version of MarkBind (eg. `v1.6.3`), you have to modify the step where `markbind-cli` is being installed to `npm i -g markbind-cli@v1.6.3`. For example, for Travis-CI, you can modifiy the `install` step in `.travis.yml` to as follows:
```
Comment (ang-zeyu) [inline docs/userGuide/tipsAndTricks.md:81]:
```suggestion
* If you want to specify a version of MarkBind (eg. `v1.6.3`), you have to modify the step where `markbind-cli` is being installed to `npm i -g markbind-cli@v1.6.3`. For example, for Travis-CI, you can modifiy the `install` step in `.travis.yml` as follows:
```

File: docs/userGuide/tipsAndTricks.md [hunk 3]
```diff
@@ -69,3 +69,48 @@ When you use links or triggers, you may encounter a situation where an unwanted
...
+  install:
+    - npm i -g markbind-cli@^1.6.3
+  ```
+
+**Setting up Netlify to use a specific version of MarkBind**
+
+Here are the steps to set up Netlify to use a specific version of MarkBind.
+
+1. Navigate to the root directory of your site.
+1. run `npm init` which will create `package.json` and `package.lock.json`
```
Comment (ang-zeyu) [inline docs/userGuide/tipsAndTricks.md:99]: Let's capitalise the first letters

**Review by ang-zeyu — COMMENTED** [review]
Couple more minor suggestions:
- remove line numbers from the code blocks
- align the images / codeblocks with the list items where applicable

**Comment by raysonkoh** [comment]
Thanks for the review comments @ang-zeyu, I have updated the PR according to them.

**Review by ang-zeyu — APPROVED** [review]
Looks really neat now, thanks! @raysonkoh

**Pull request merged**